kolibri/plugins/perseus_viewer/assets/src/views/PerseusRendererIndex.vue

```diff
@@ -15,7 +15,7 @@
           />
         </div>
         <KGrid>
-          <KGridItem :layout="{ span: 8 }">
+          <KGridItem :layout="{ span: 4 }">
```

Comment:
This does seem to produce quite a narrow display on a fairly wide screen from your animation - maybe we should conditionalize these values for different aspect ratios?

Reply:
I've revised the layouts now to go `:layout12={ span: 8 }` to get a ~66% width on larger screens. Then the default to 100% on medium or smaller which seems to avoid having too-squished on tablet & mobile without having it too wide on a larger screens.

The below video goes from ~1100px wide down to like ~200px wide at smallest. I then show it with the hints enabled because the layout affects it.

Previously the `:layout="{ span: 4 }"` would be 4/12 at large, 4/8 at tablet and 4/4 at mobile - so when it was set to `:layout="{span: 8}"` it complained about the span exceeding the num cols available (4 at mobile size).

https://user-images.githubusercontent.com/6356129/142263715-472d38de-2fdf-46ca-b00f-b2c1d909005b.mp4